week1/index.html

```diff
@@ -0,0 +1,140 @@
+  </head>
+  <body>
+    <!-- 네비게이션 -->
+    <nav class="navi">
+      <div class="left">
+        <div><img src="./img/판다 얼굴.svg" alt="판다 얼굴 로고" /></div>
+        <p class="pandaLetter">판다마켓</p>
+      </div>
+      <a href="./loginpage/login.html">
+      <h1 class="loginButton click">
+        로그인
+      </h1></a>
+    </nav>
+
+    <!-- 헤더 -->
+    <header>
+      <div class="headerText">
+        <p class="bigText">
+          일상의 모든 물건을 <br />
+          거래해보세요
+        </p>
+        <a href="/items"><h2 class="click headerButton">구경하러 가기</h2></a>
+      </div>
+      <div>
+        <img src="./img/Img_home_top.png" alt="가방들고 혼자 인사하는 판다" />
+      </div>
+    </header>
+
+    <!-- 메인 -->
+    <main>
+      <!-- 메인 섹션 1 -->
+      <section class="box">
+        <div>
+          <img
+            src="./img/Img_home_01.png"
+            alt="인기상품, 옷을 구경하는 판다"
+            class="centerImage"
+          />
+        </div>
+        <div class="centerText">
+          <p class="blue">Hot item</p>
+          <div>
+            <p class="bigText">
+              인기 상품을 <br />
+              확인해 보세요
+            </p>
+            <p class="smallText">
+              가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <!-- 메인 섹션 2 -->
+      <section class="box middle">
+        <div class="centerText second">
+          <p class="blue">Search</p>
+          <div>
+            <p class="bigText">구매를 원하는<br />상품을 검색하세요</p>
+            <p class="smallText">
+              구매하고 싶은 물품은 검색해서 <br />쉽게 찾아보세요
+            </p>
+          </div>
+        </div>
+        <img
+          src="./img/Img_home_02.png"
+          alt="상품 검색 이미지"
+          class="centerImage"
+        />
+      </section>
+
+      <!-- 메인 섹션 3 -->
+      <section class="box">
+        <div>
+          <img
+            src="./img/Img_home_03.png"
+            alt="상품등록 파일 이미지"
+            class="centerImage"
+          />
+        </div>
+        <div class="centerText">
+          <p class="blue">Register</p>
+          <div>
+            <p class="bigText">판매를 원하는 <br />상품을 등록하세요</p>
+            <p class="smallText">
+              어떤 물건이든 판매하고 싶은 상품을 <br />쉽게 등록하세요
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <!-- 바텀 -->
+      <div class="bottom">
+        <p class="bigText">믿을 수 있는<br />판다마켓 중고 거래</p>
+        <div>
+          <img
+            src="./img/Img_home_bottom.png"
+            alt="둘이서 대화하고 인사하는 판다"
+          />
+        </div>
+      </div>
+    </main>
+
+    <!-- 푸터 -->
+    <footer class="foot">
+      <p class="click">©codeit - 2024</p>
+      <div class="PolicyFQA">
+        <p><a href="/Privacy">Privacy Policy</a></p>
+        <p><a href="/FAQ">FAQ</a></p>
+      </div>
+      <!-- 푸터 sns -->
+      <div class="snsLink click">
+        <div>
+          <a href="facebook"
+            ><img src="./img/ic_facebook.png" alt="facebook"
+          /></a>
```

Comment:
```suggestion
          <a href="facebook">
            <img src="./img/ic_facebook.png" alt="facebook"/>
          </a>
```

이렇게 쓰는게 조금 더 가독성 있어보이고 좋을 것 같습니다 ㅎㅎ